nextflow.config

```
@@ -27,6 +28,7 @@ params {
     // HiGlass options
     upload_higlass_data             = false
     higlass_upload_directory        = null
+    higlass_data_project_dir            = null
```

The so-called Harshil alignment
```suggestion
    higlass_data_project_dir        = null
```